frontend/components/ui/login-card.tsx

```
@@ -0,0 +1,118 @@
+                    <span>or</span>
+                </div>
+                <form className='account__form' onSubmit={loginHandler}>
+                    <div className='form-grp'>
+                        <label htmlFor='email'>Email</label>
+                        <input id='email' name='email' type='text' placeholder='Email' />
+                        <p className='form_error'></p>
+                    </div>
+                    <div className='form-grp'>
+                        <label htmlFor='password'>Password</label>
+                        <input id='password' name='password' type='password' placeholder='Password' />
+                        <p className='form_error'></p>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)